docs/reference.rst

```diff
@@ -21,6 +25,18 @@ Tasks
     :members: defer, configure
 
 
+Builtin tasks
+-------------
+
+Procrastinate has builtin tasks that are all available from the CLI.
+
+Clean old jobs
+^^^^^^^^^^^^^^
+
+This task lets you delete old jobs and linked events from the database. For more
+information, see :ref:`the how-to section <clean_old_jobs>`.
```

Comment:
This is the reference doc, we should document the signature of the task, no ?

Reply:
You may want to do this the other way around: a quick presentation in the how-to section, pointing here, and the details of the signature here.

Note: you can still make it look like other functions, either with [this](https://stackoverflow.com/a/30584552) trick or by using a plain [`py:function`](https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html?highlight=method#directive-py:function)